core/src/main/java/com/adobe/aio/workspace/Workspace.java

```diff
@@ -254,6 +256,10 @@ public Builder clientSecret(final String clientSecret) {
         jwtbuilder = JwtContext.builder();
       }
       jwtbuilder.clientSecret(clientSecret);
+      if(oAuthBuilder == null){
```

Comment:
let's remove this new lines 259-262 ... 
clientSecret() method is to be deprecated and was only kept for jwt backward compatibilty 
we could mark is as such

Reply:
Review Comments Incorporated